### PR TITLE
[IS-16408] Adding connector-config documentation

### PR DESCRIPTION
**Pull request by branislavjenco**

### CODE REVIEW

File: hub/documentation/data-synchronization/connectors/configuration-connectors.rst [hunk 1]
```diff
@@ -0,0 +1,282 @@
+=======================
+Connector Configuration
+=======================
+
+In a particular environment, connectors can be configured using the ``config.json`` file found in the `sesam-talk-config <https://github.com/datanav/sesam-talk-config/>`_ repository. This file has a property called ``connector_config`` whose value is the configuration of an embedded pipe with the same name. By edititng the entities in the embedded source, the various connectors can be configured.
```
Comment (grove) [inline hub/documentation/data-synchronization/connectors/configuration-connectors.rst:5]: Typo: edititng

File: hub/documentation/data-synchronization/connectors/configuration-connectors.rst [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+     - Array
+     - A list of connectors that can be connected to when this system is the vendor. This filters the connectors that can be seen in the dashboard after the first connection. The entries in the list need to correspond to the _ids of the other connectors.
+     - No
+
+   * - ``vendor.hidden_entity_type_options``
+     - Array
+     - List of entity type options that should be hiddne from the UI for all connectors, when we are using this vendor. Example values are ``fullsync`` and ``share_enabled``.
+     - No
+
+   * - ``vendor.homepageLogo``
```
Comment (grove) [inline hub/documentation/data-synchronization/connectors/configuration-connectors.rst:202]: Why is this one camel-case? 👮

File: hub/documentation/data-synchronization/connectors/configuration-connectors.rst [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+   :widths: 10, 10, 60, 10
+
+   * - ``auth.type``
+     - "api_key" | "oauth2"
+     - Specifies the type of authorization - either ``oauth2`` for when connectors support the OAuth protocol or `api_key` for other approaches based on manually providing api_keys, tokens and similar.
+     - Yes
+
+   * - ``auth.api_base_url``
+     - String
+     - Similar to the config for the API key type authentication, the connector-deployer uses this for the {{@ base_url @}} value. In the Management console it is only needed for the Tripletex connector.
```
Comment (grove) [inline hub/documentation/data-synchronization/connectors/configuration-connectors.rst:256]: Make `{{@ base_url @}} `a literal maybe?

**Review by grove — APPROVED** [review]

File: hub/documentation/data-synchronization/connectors/configuration-connectors.rst (hunk 1 above)
Comment (branislavjenco) [reply hub/documentation/data-synchronization/connectors/configuration-connectors.rst]: Fixed

File: hub/documentation/data-synchronization/connectors/configuration-connectors.rst (hunk 3 above)
Comment (branislavjenco) [reply hub/documentation/data-synchronization/connectors/configuration-connectors.rst]: Fixed

File: hub/documentation/data-synchronization/connectors/configuration-connectors.rst (hunk 2 above)
Comment (branislavjenco) [reply hub/documentation/data-synchronization/connectors/configuration-connectors.rst:202]: That's how it is right now, but I agree we should change it.

**Pull request merged**